packages/email/mailer.ts

```diff
@@ -46,10 +46,14 @@ const getTransport = () => {
     host: process.env.NEXT_PRIVATE_SMTP_HOST ?? 'localhost:2500',
     port: Number(process.env.NEXT_PRIVATE_SMTP_PORT) || 587,
     secure: process.env.NEXT_PRIVATE_SMTP_SECURE === 'true',
-    auth: {
-      user: process.env.NEXT_PRIVATE_SMTP_USERNAME ?? '',
-      pass: process.env.NEXT_PRIVATE_SMTP_PASSWORD ?? '',
-    },
+    ignoreTLS: process.env.NEXT_PRIVATE_SMTP_SECURE === 'false' ? true : undefined,
+    auth:
+      process.env.NEXT_PRIVATE_SMTP_USERNAME && process.env.NEXT_PRIVATE_SMTP_USERNAME !== ''
```

Comment:
The 	`&&` is redundant since an empty string is falsey 😄 

<img width="261" alt="image" src="https://github.com/documenso/documenso/assets/13398220/701fda7c-9cb7-4e68-bdf0-2168407b93c3">

Reply:
done